packages/amplify-cli/src/initialize-env.ts

```diff
@@ -76,10 +76,10 @@ export const initializeEnv = async (
     } catch (e) {
       printer.error(`Could not initialize '${currentEnv}': ${e.message}`);
       context.usageData.emitError(e);
-      process.exit(1);
-    } finally {
       context.usageData.stopCodePathTimer(ManuallyTimedCodePath.INIT_ENV_PLATFORM);
+      exitOnNextTick(1);
```

Comment:
What is `exitOnNextTick` for?

Reply:
It calls process.exit(), but first closes the logger:
https://github.com/aws-amplify/amplify-cli/blob/dev/packages/amplify-cli-core/src/exitOnNextTick.ts